driver/src/price_finding/naive_solver.rs

```diff
@@ -145,6 +160,7 @@ impl PriceFinding for NaiveSolver {
                     }
                     None => continue,
                 }
+                matching_orders_indexes = Some((i, j));
```

Comment:
Sorry to be a pain, but I really think we should be using the term `indices` here

Reply:
No problem. I agree with using use proper grammar and spelling in the code.